src/Footer.jsx

```diff
@@ -23,7 +23,11 @@ const Footer = (props) => (
       <p>© The New York Public Library, {new Date().getFullYear()}</p>
     </div>
     <span className="logoText icon-logo-type">
-      <LionLogoWithText title="New York Public Library" ariaHidden={false} />
+      <LionLogoWithText
+        title="New York Public Library"
+        ariaHidden={false}
+        focusable="false"
```

Comment:
The value of focusable that we have assigned is a boolean, so it might complain if we give it a string here. However, the issue is actually deeper.

focusable is a enumerated attributes, yet, looks like React can take either boolean or string for it. Here's the discussion
https://github.com/facebook/react/issues/6212

I think either way is fine, string or boolean. We just need to keep it consistent.

Reply:
Updated. It did not complain. Do you run and tests the PRs yourself or just check the code?

Reply:
No, it _should_ be boolean since that's what we defined in `@nypl/dgx-svg-icons`, even if react is okay with both.

Reply:
What I meant by either way is that if you wanted to use string, we should go back to change the type to string in `@nypl/dgx-svg-icons`.

I didn't run eslint for that, because just by seeing the code first, I know I should comment about the issue with you already.

Reply:
Alright, you did say it might complain so I assumed you didn't check since you weren't sure. Keeping it as boolean for now is fine. String was probably the better choice since the boolean gets rendered as a string anyway by react.